### PR TITLE
Playroom: Bind state via 'stateName' prop, add 'toggleState'

### PR DESCRIPTION
**Pull request by markdalgleish**
Also updated the prototyping guide to use these new APIs.

### CODE REVIEW

File: lib/playroom/playroomState.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+  const resolvedValue =
+    value ??
+    (stateKey
+      ? playroomState.getState(stateKey) ?? defaultValue
+      : internalStateValue);
+
+  return stateKey
+    ? [resolvedValue, handleChange]
+    : [resolvedValue, handleChange];
```
Comment (michaeltaranto) [inline lib/playroom/playroomState.tsx:121]: Let's flatten this one out 😉

File: sku.config.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = {
   playroomFrameComponent: './lib/playroom/FrameComponent.tsx',
   playroomScope: './lib/playroom/useScope.ts',
   playroomTarget: './site/dist/playroom',
-  playroomWidths: [320, 768, 1024, 1400],
+  playroomWidths: [320, 739, 768, 1024, 1400],
```
Comment (michaeltaranto) [inline sku.config.js:42]: Debugging relic?

File: lib/playroom/playroomState.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
+import React, { useContext, ReactNode } from 'react';
+import { createContext, useState } from 'react';
```
Comment (mattcompiles) [inline lib/playroom/playroomState.tsx:2]: Two react imports in the same file

File: sku.config.js (hunk 2 above)
Comment (markdalgleish) [reply sku.config.js]: Oops. Yep.

**Review by michaeltaranto — APPROVED** [review]
Exciting 👏  🚀

**Pull request merged**